src/tests/test.spec.js

```diff
@@ -0,0 +1,31 @@
+import { expect } from '@playwright/test';
+import { test } from '../fixtures/base';
+import { users } from '../test-data/users';
+
+test.describe('Inventory Sorting Tests', () => {
+    test.beforeEach(async ({ app }) => {
+        const { username, password } = users.standardUser;
+        await app.login.navigate();
+        await app.login.performLogin(username, password);
+    });
+
+    test('Sort items by different criteria', async ( /** @type {{ app: import('../pages/Application').Application }} */ { app }) => {
```

Comment:
I would you use Data Driven Testing approach in this case. Please watch this video https://www.youtube.com/watch?v=7INsJsP6orY&ab_channel=qa%D1%81%D0%B5%D0%BC%D0%BF%D0%B0%D0%B9%7C%D0%B0%D0%B2%D1%82%D0%BE%D0%BC%D0%B0%D1%82%D0%B8%D0%B7%D0%B0%D1%86%D1%96%D1%8F%D1%82%D0%B5%D1%81%D1%82%D1%83%D0%B2%D0%B0%D0%BD%D0%BD%D1%8F

Reply:
There should be 4 different tests.